app/models/web_money_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class WebMoneyDecorator
```

Comment:
I think this class name needs namespace for avoid conflict :santa: 

e.g. `SpreeKomoju::WebmoneyDecorator`.